api_tests/regtest/alice/default.toml

```diff
@@ -5,10 +5,13 @@ secret_seed = "f87165e305b0f7c4824d3806434f9d0909610a25641ab8773cf92a48c9d77670"
 [http_api]
 address="0.0.0.0"
 port=8000
-logging=true
+
+[comit_i]
```

Comment:
Or maybe add it to Bob's and Charlie's configs if you want these to be complete config files.

Reply:
1. We do for the cross-origin auth
2. It's actually good to have it, it allows you to quickly start a node with comit-i using `COMIT_NODE_CONFIG_PATH=./api_tests/regtest/alice/ cargo run --package comit_node`

Reply:
I see now! I like that then. Please add it to the other configs though :wink:

Reply:
After discussion can we:

1. Remove this from here
2. Have COMIT-i tests in its own directory and have a `config.toml` that sets the config dir to something specific to COMIT-i tests (i.e. not `./api_tests/regtest/alice/`)

To make the above work you'll need to change the harness so it doesn't run tests from `api_tests` as it's cwd. This will cause some collateral damage in various places.

Reply:
The 🚀 is here to show that the scope creep is heading to the moon.